kratos/geometries/nurbs_surface_geometry.h

```diff
@@ -191,6 +191,24 @@ class NurbsSurfaceGeometry : public Geometry<typename TContainerPointType::value
         return Kratos::make_shared<NurbsSurfaceGeometry>(ThisPoints);
     }
 
+    ///@}
+    ///@name Information
+    ///@{
+
+    /// Returns number of points per direction.
+    SizeType PointsNumberInDirection(IndexType LocalDirectionIndex) const override
+    {
+        if (LocalDirectionIndex == 0) {
+            return this->NumberOfControlPointsU();
+        }
+        else if (LocalDirectionIndex == 1) {
+            return this->NumberOfControlPointsV();
+        }
+        KRATOS_ERROR_IF(LocalDirectionIndex > 2)
+            << "Possible direction index in NurbsSurfaceGeometry reaches from 0-2. Given direction index: "
+            << LocalDirectionIndex << std::endl;
```

Comment:
Same comment? The behaviour is undefined if the user provides `LocalDirectionIndex = 2`?